### PR TITLE
Add `Tag::addClass()` and deprecate `Tag::class()`

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #123

### CODE REVIEW

File: src/Tag/Base/Tag.php [hunk 1]
```diff
@@ -92,8 +92,23 @@ final public function id(?string $id): self
      * @param string|null ...$class One or many CSS classes.
      *
      * @return static
+     *
+     * @deprecated Use {@see addClass()} or {@see replaceClass} instead. Please note that in next major
+     * version method `replaceClass()` will be renamed to `class()`.
      */
     final public function class(?string ...$class): self
```
Comment (terabytesoftw) [inline src/Tag/Base/Tag.php:99]: Why not string `btn btn-lg text-primary` string is enough, why do we need an array ?

Comment (vjik) [reply src/Tag/Base/Tag.php:99]: > Why not string `btn btn-lg text-primary` string is enough, why do we need an array ?

Method accept both string and array.

File: src/Tag/Base/Tag.php [hunk 2]
```diff
@@ -92,8 +92,23 @@ final public function id(?string $id): self
      * @param string|null ...$class One or many CSS classes.
      *
      * @return static
+     *
+     * @deprecated Use {@see addClass()} or {@see replaceClass} instead. Please note that in next major
```
Comment (arogachev) [inline src/Tag/Base/Tag.php:96]:
```suggestion
     * @deprecated Use {@see addClass()} or {@see replaceClass} instead. Please note that in the next major
```

File: src/Tag/Base/Tag.php [hunk 3]
```diff
@@ -92,8 +92,23 @@ final public function id(?string $id): self
...
+     * version method `replaceClass()` will be renamed to `class()`.
```
Comment (arogachev) [inline src/Tag/Base/Tag.php:97]:
```suggestion
     * version `replaceClass()` method will be renamed to `class()`.
```

File: src/Tag/Base/Tag.php (hunk 2 above)
Comment (arogachev) [inline src/Tag/Base/Tag.php:96]:
```suggestion
     * @deprecated Use {@see addClass()} or {@see replaceClass()} instead. Please note that in next major
```

**Review by arogachev — CHANGES_REQUESTED** [review]
:+1:

**Review by arogachev — APPROVED** [review]

**Review by samdark — APPROVED** [review]

**Pull request merged**